Description: Fix database upgrade to 1.13.0.
 error encountered processing /usr/share/dbconfig-common/data/icinga-idoutils/upgrade/pgsql/1.13.0:
 ERROR: constraint "uq_servicedependencies" of relation "icinga_servicedependencies" does not exist
Author: Bas Couwenberg <sebastic@debian.org>
Bug-Debian: https://bugs.debian.org/861077
Forwarded: https://github.com/Icinga/icinga-core/pull/1595

--- a/module/idoutils/db/pgsql/upgrade/pgsql-upgrade-1.13.0.sql
+++ b/module/idoutils/db/pgsql/upgrade/pgsql-upgrade-1.13.0.sql
@@ -11,8 +11,11 @@
 -- #7765 drop unique constraint
 -- -----------------------------------------
 
-ALTER TABLE icinga_servicedependencies DROP CONSTRAINT uq_servicedependencies;
-ALTER TABLE icinga_hostdependencies DROP CONSTRAINT uq_hostdependencies;
+ALTER TABLE icinga_servicedependencies DROP CONSTRAINT IF EXISTS icinga_servicedependencies_instance_id_key;
+ALTER TABLE icinga_hostdependencies DROP CONSTRAINT IF EXISTS icinga_hostdependencies_instance_id_key;
+
+ALTER TABLE icinga_servicedependencies DROP CONSTRAINT IF EXISTS UQ_servicedependencies;
+ALTER TABLE icinga_hostdependencies DROP CONSTRAINT IF EXISTS UQ_hostdependencies;
 
 CREATE INDEX idx_servicedependencies ON icinga_servicedependencies(instance_id,config_type,service_object_id,dependent_service_object_id,dependency_type,inherits_parent,fail_on_ok,fail_on_warning,fail_on_unknown,fail_on_critical);
 CREATE INDEX idx_hostdependencies ON icinga_hostdependencies(instance_id,config_type,host_object_id,dependent_host_object_id,dependency_type,inherits_parent,fail_on_up,fail_on_down,fail_on_unreachable);
